src/Microsoft.DotNet.Darc/DarcLib/Remote.cs

```diff
@@ -181,20 +181,20 @@ public async Task<List<GitFile>> CommitUpdatesAsync(
         SemanticVersion targetDotNetVersion = null;
         var mayNeedArcadeUpdate = arcadeItem != null && repoUri != arcadeItem.RepoUri;
         // If we find version files in src/arcade, we know we're working with a VMR
-        bool sourceRepoIsVmr = true;
+        string mapping = VmrInfo.ArcadeMappingName;
```

Comment:
Same comment about using string as a boolean

Reply:
added the boolean back